app/controllers/projects_controller.rb

```diff
@@ -29,7 +29,7 @@ def index
           @featured_partners = SitePartner.featured
           @regular_partners = SitePartner.regular
           @site_partners = @featured_partners + @regular_partners
-          @channels = Channel.visible
+          @channels = visible_channels
```

Comment:
What do you think of creating two scenarios of tests, to check if the variable `@channels` got the correct value?